agenta-web/src/components/Playground/ViewNavigation.tsx

```diff
@@ -66,13 +78,51 @@ const ViewNavigation: React.FC<Props> = ({
             variantContainerPath()
         }
 
+        const restartContainerHandler = async () => {
+            // Set restarting to true
+            setRestarting(true)
+
+            // Set payload to send to backend
+            const data: RestartVariantDocker = {
+                app_name: appName.toLowerCase(),
+                variant_name: variant.variantName,
+            }
+            try {
+                const response = await restartAppVariantContainer(data)
+                if (response.status === 200) {
+                    showNotification({
+                        type: "success",
+                        message: "App Container",
+                        description: `${response.data.message}`,
+                        duration: 5,
+                        key: response.status,
+                    })
+
+                    // Set restarting to false
+                    setRestarting(false)
+                    setTimeout(() => {
+                        // Wait for 3s before reloading component
+                        router.reload()
+                    }, 3000)
+                }
+            } catch (err: any) {
+                if (axios.isAxiosError(err) && err.response?.status === 500) {
```

Comment:
why are we checking for this specific error condition to set restarting state to false?

Reply:
Because of this error:

`Type error: Property 'response' does not exist on type 'Error'.`